lms/urls.py

```diff
@@ -91,6 +91,8 @@
     # independently of courseware. https://github.com/edx/edx-val
     url(r'^api/val/v0/', include('edxval.urls')),
 
+    # Full Course/Library Import/Export API
+    url(r'^api/import_export/v1/', include('openedx.core.djangoapps.import_export.urls')),
```

Comment:
I'm not super familiar with those scopes (and what is in them we would be missing), but we are already importing courses on the LMS with the sysadmin dashboard course import feature, which has been in for a really long time, this would just expose that as a more easily automatable endpoint.